### PR TITLE
[FEATURE] added redoc plugin

### PR DESCRIPTION
**Pull request by barthy-koeln**
Hi Jan,

Thanks for the fantastic plugin.

I've added a [redoc](https://github.com/Redocly/redoc) plugin because that's my preferred UI.
Others might also prefer it.

Barthy

### CODE REVIEW

File: src/redocPlugin.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                    margin: 0;
+                    padding: 0;
+                  }
+                </style>
+              </head>
+              <body>
+                <redoc spec-url="${req.protocol}://${req.header(
+            'host',
+          )}/api${specEndpoint}"></redoc>
+                <script src="https://cdn.redoc.ly/redoc/latest/bundles/redoc.standalone.js"> </script>
```
Comment (Teyras) [inline src/redocPlugin.ts:44]: The rest of the API docs plugins use fixed versions from unpkg.com. Did you consider using that, or is this just the recommended way of deploying ReDoc?

I'm not opposed to leaving it as is, just curious about your thoughts :slightly_smiling_face:

**Review by Teyras — APPROVED** [review]
Whoa, thanks for your contribution!

File: src/redocPlugin.ts (hunk 1 above)
Comment (barthy-koeln) [reply src/redocPlugin.ts:44]: It's the basic example from the redoc readme. I'm impartial to which CDN to use, but would only limit the major version number (expecting people to follow the semantic versioning constraints :smile: )

The unpkg CDN would be: `https://unpkg.com/redoc@^2/bundles/redoc.standalone.js`

Shall I commit that?

Comment (Teyras) [reply src/redocPlugin.ts:44]: Yeah, let's do that for the sake of consistency.

Comment (barthy-koeln) [reply src/redocPlugin.ts]: Done. I also removed the Google fonts CDN.

**Comment by Teyras** [comment]
@barthy-koeln thanks again for the PR, it should be released on npm now.

**Pull request merged**